.travis.yml

```diff
@@ -15,4 +15,8 @@ before_script:
 script:
 - export NODE_PATH=$TRAVIS_BUILD_DIR'/privly.safariextension/privly-applications/test/node_modules'
 - cd test
-- ./run_each.sh ../privly.safariextension/privly-applications/test/karma.conf-ci.js
+- "if [ ${TRAVIS_PULL_REQUEST} = 'false' ]; then ./run_each.sh ../privly.safariextension/privly-applications/test/karma.conf-ci.js; fi" #Run the unit tests
+- "cd ../privly.safariextension/privly-applications/test"
+- "if [ ${TRAVIS_PULL_REQUEST} = 'false' ]; then mv coverage/* coverage/coverage.lcov; fi" # Rename the coverage results file
```

Comment:
You can remove these lines and change the one below to `... cat ./privly.safariextension/privly-applications/test/coverage/*.lcov ...`. I am nearly indifferent on this change, so if you have a preference for renaming it, I can leave it as-is.